index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
+    />
+    <link rel="stylesheet" href="./index.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200&display=swap" rel="stylesheet">
+
+</head>
+  <body>
+    <header>
+     <nav> 
+    <img src="./Images/Hamburger_icon.svg.png" class="hamburger" alt="" />
+      <ul class="menu-items">
+        <img src="./Images/Cross-Icon.png" class="Cross-Icon" alt="" />
+        <li class="menu-item-mobile">
+          <a href="./about.html">About</a>
+        </li>
+        <li class="menu-item-mobile">
+          <a href="#">Program</a>
+        </li>
+        <li class="menu-item-mobile">
+          <a href="#">Join</a>
+        </li>
+        <li class="menu-item-mobile">
+          <a href="#">Sponsor</a>
+        </li>
+        <li class="menu-item-mobile">
+          <a href="#">Help</a>
+        </li>
+      </ul>
+
+      <div class="myLinks">
+        <ul>
+          <li><a href="www.facebook.com"></a> <img src="/Images/fb.icon.png" alt=""></li>
+          <li><a href="www.gooogle.com"></a><img src="/Images/Twttr-logo.png" alt=""></li>
+          <li><a href="/index.html">Login</a></li>
+          <li><a href="about.html">MyPage</a></li>
+          <li><a href="#">English</a></li>
+        </ul>
+      </div> 
+      
+      <ul class="nav-list">
+        <li class="menu-item">
+          <img src="./Images/My-Logo.png" alt="Logo">
+        </li>
+        <li class="menu-item">
+          <a href="./about.html">About</a>
+        </li>
+        <li class="menu-item">
+          <a href="#">Program</a>
+        </li>
+        <li class="menu-item">
+          <a href="#">Join</a>
+        </li>
+        <li class="menu-item">
+          <a href="#">Sponsor</a>
+        </li>
```

Comment:
- [ ] **Good job** :+1: :+1: Implementing the required changes by the previous reviewer, :clap: 
According to the previous reviewers, Kindly try to improve your ```main nav bar``` design by adjusting space between and around the content as requested and it would be better to use the exact background.

_Remember it is important that you try to replicate the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as closely as possible to give confidence to recruiters in your ability to produce exact business requirements :pray:_